server/submission/definitions.js

```diff
@@ -275,22 +280,29 @@ const manuscriptSchema = Joi.object()
       })
       .required(),
     suggestedSeniorEditors: Joi.array()
-      .items(Joi.string().required())
+      .items(Joi.number().required())
```

Comment:
Not sure why these are now numbers and not strings?

Reply:
It used to be a name but now it's a numerical ID. I guess it will be a UUID eventually so perhaps I should just cast the numbers to strings?

Reply:
Lets just keep an eye on this for now - as you say it will need to be changed again to validate for UUID's ... have you seen this which is quite recent? https://github.com/olistic/graphql-type-uuid